src/formElementPlugins/checkbox/src/ApplicationView.tsx

```diff
@@ -199,14 +256,54 @@ export const getCheckboxStructure = (
         selected: cb?.[checkboxPropertyNames.selected] || false,
       }
   })
+
+  if (hasOther)
+    checkboxElements.push({
+      label: otherLabel,
+      text: initValues ? initValues.other?.text ?? '' : '',
```

Comment:
Am I understanding right that a default value for other - or is this just calling "PLACEHOLDER_OTHER" from the strings json?

Reply:
It's just creating an additional checkbox (on top of the defined ones) if the "hasOther" option parameter is turned on.